libs/internal/include/launchdarkly/context_filter.hpp

```diff
@@ -34,6 +34,17 @@ class ContextFilter {
      */
     JsonValue filter(Context const& context);
 
+    /**
+     * Filter the given context and produce a JSON value. If a provided context
+     * is anonymous, all attributes are redacted.
+     *
+     * Only call this method for valid contexts.
+     *
+     * @param context The context to redact.
+     * @return JSON suitable for an analytics event.
+     */
+    JsonValue filter_with_anonymous_redaction(Context const& context);
```

Comment:
Wasn't sure what the casing convention was for this repo so don't hesitate to nitpick!